src/test/resources/Features/OCD-2873-SimplifiedEditingOfDeveloper.feature

```diff
@@ -0,0 +1,64 @@
+@Regression
+Feature: OCD-2873 Simplified Editing of Developer
+  This test checks whether edit, split and merge developer functionality work as expected
+
+Scenario Outline: User logs in as ROLE and sees developer edit link exists on new developer page
+    Given I'm logged in as "<ROLE>"
+    When I navigate to the developer edit page for developer "1854"
+    Then I see developer edit link for developer "1854"
+    Examples:
+      |ROLE       |
+      |ROLE_ACB   |
+      |ROLE_ADMIN |
+      |ROLE_ONC   |
```

Comment:
Could we also have a scenario for a user who is not logged in at all and test that they don't see the edit/merge/split links?